### ISSUE TITLE
plc5

### ISSUE THREAD

**Issue by gilesbradshaw** [cmseaton42/node-ethernet-ip#47]
would this work with plc5s?

**Comment by jhenson29** [comment]
I don't think so. I think it only works with ControlLogix and CompactLogix right now. I didn't think PLC5 even had EIP, but I did a quick google search and saw that it does?

**Comment by gilesbradshaw** [comment]
yes with a card you can bridge ethernet ip to dh+

**Comment by cmseaton42** [comment]
That moment when you accidently hit the close issue button lol

**Comment by cmseaton42** [comment]
Hey @gilesbradshaw, there currently isnt support for PLC5.  There is another library out in the node world that may help you that implements cpppo that may be of use to you but I am unable to locate it at the moment :(.  The library is written to reference the specific EIP Tag Instances in the more recent Rockwell controllers.

**Comment by gilesbradshaw** [comment]
Ok thanks - maybe I have a play with it anyhow - I wonder how hard it could be to adapt

**Comment by jhenson29** [comment]
This? https://www.npmjs.com/package/nodepccc
